llvm/include/llvm/LinkAllPasses.h

```diff
@@ -200,10 +201,11 @@ namespace {
       (void) llvm::createMergeFunctionsPass();
       (void) llvm::createMergeICmpsLegacyPass();
       (void) llvm::createExpandMemCmpPass();
-      (void)llvm::createSYCLLowerWGScopePass();
-      (void)llvm::createSYCLLowerESIMDPass();
-      (void)llvm::createESIMDLowerLoadStorePass();
-      (void)llvm::createESIMDLowerVecArgPass();
+      (void) llvm::createSYCLLowerWGScopePass();
+      (void) llvm::createSYCLLowerESIMDPass();
+      (void) llvm::createESIMDLowerLoadStorePass();
+      (void) llvm::createESIMDLowerVecArgPass();
```

Comment:
please avoid redundant whitespace changes

Reply:
I was a bit confused: should I follow general code style in the file, or should I follow surrounding SYCLlish style? So I decided to just align them, does it make sense?

Reply:
In general, you should properly format only your changes, not touching existing. There can be exceptions, of course. @bader, do you think it is OK to in this case?

Reply:
Ideally this change should go as a separate patch.

Reply:
okay